generated/nifake/unit_tests/test_session.py

```diff
@@ -387,6 +415,47 @@ def test_multiple_array_types(self):
                 matchers.ViInt16BufferMatcher(input_array_of_integers)
             )
 
+    '''
```

Comment:
why is this commented out?

Reply:
Removed. It was left over from when I was fixing the test